app/src/interfaces/file-image/file-image.vue

```diff
@@ -185,6 +185,10 @@ async function imageErrorHandler() {
 	}
 }
 
+function onUpload(image: any) {
+	if (image) update(image.id);
```

Comment:
```suggestion
	if (image?.id) update(image.id);
```
To be on the safe side should we check for the actual ID? 🤔

Reply:
In this case `image` is typed as `any`, so we have to assume that it might be.. `any`thing 🙂